app/utils/importFile.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import Document from '../models/Document';
+import DocumentsStore from '../stores/DocumentsStore';
+
+type Options = {
+  file: File,
+  documents: DocumentsStore,
+  collectionId: string,
+  documentId?: string,
+};
+
+const importFile = async (
+  { documents, file, documentId, collectionId }: Options,
+  callback: Document => *
```

Comment:
Why not return a promise? We dont use callbacks anywhere

Reply:
Because I'm a lazy bum. Will update 😄